communitiesuk/funding-service-design-post-award-docker-runner
# Add celery auto refresh with watchfiles

### Change description
Add celery auto reload with `watchlists`.

### How to test
Pull `watchlists` dependency [branch](https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/761)
Check refresh by editing files after turning it on
Run e2e tests on local
